Woof/Woof/ViewModel/SitterProfileViewModel.swift

```diff
@@ -46,6 +50,22 @@ final class SitterProfileViewModel: ObservableObject {
             .save(data, for: KeyValueStorage.Key.currentSitter)
     }
 
+    /// Uploads the pet sitter's data to the backend server asynchronously.
+    ///
+    /// `isErrorOccurred` is set to `false`, indicating no errors.
+    /// If there is a network error during the upload,`isErrorOccurred` is set to `true`.
+    func upload() async throws {
+        do {
+            let endpoint = WoofAppEndpoint.addNewSitter(currentSitter.asDictionary())
+            _ = try await networkService.request(endpoint)
```

Comment:
Why do we need to store the instance of the `NetworkService`?
```suggestion
            _ = try await NetworkService().request(endpoint)
```

Reply:
Yes you're right, it doesn't make sense, fixed with 06c4f69